resources/views/users/ldap.blade.php

```diff
@@ -63,7 +68,7 @@
           </tr>
 
           @foreach (Session::get('summary') as $entry)
-          <tr {!! ($entry['status']=='success') ? 'class="success"' : 'class="danger"' !!}>
```

Comment:
I'm not sure if we absolutely need the re-casing of this, but if so then it's fine. If not I'd rather not see it in this PR.

Reply:
I changed it because the status text in the sync command returns uppercase.